sources/p987.6/sections/03-defs_and_terms.adoc

```diff
@@ -0,0 +1,67 @@
+
+== Terms and definitions
+
+=== Definitions
+
+For the purposes of this document, the following terms and definitions
+apply. The IEEE Standards _Dictionary Online_ should be consulted
+for terms not defined in this clause. footnote:[The _IEEE Standards
+Dictionary Online_ is available at: http://dictionary.ieee.org.]
+
+
+==== acceleration-insensitive drift rate
+related:seealso[drift rate]
+related:seealso[systematic drift rate]
+
+The component of...
+
+==== code set
+related:see[coded character set]
+
+==== coded character set
+related:equivalent[code set]
+
+A set of characters.
+
+==== drift rate
+
+The slope at a stated time of...
+
+[.source]
+<<iso-iec_9945-1>>
+
+==== input reference axis (IRA)
```

Comment:
"IRA" needs to be encoded as an "initialism", even though it is not yet showing up correctly.

Reply:
I've re-encoded it like:
````
==== input reference axis
alt:[IRA]
````

Reply:
By `initialism` I really meant using the hard-to-use `abbreviation-type` attribute. 😉 

The semantically correct way of doing it is:
```adoc
==== input reference axis

[%metadata]
abbreviation-type:: initialism

preferred:[IRA]
```

Using `preferred:[...]` here because it is not an "alternate/admitted term", it is an equally "preferred" term. I know the rendering isn't correct yet but that's a separate issue.

Reply:
Ok! Changes applied.

Reply:
Thanks @manuelfuenmayor !